src/include/distributed/backend_data.h

```diff
@@ -54,6 +54,7 @@ extern void UnSetDistributedTransactionId(void);
 extern void UnSetGlobalPID(void);
 extern void AssignDistributedTransactionId(void);
 extern void AssignGlobalPID(void);
+extern void SetTransactionIdClockValue(uint64 transactionClockValue);
```

Comment:
do we really need to export this function ?

Reply:
We are calling it here

`remote_transaction.c SetTransactionClusterClock     1553 SetTransactionIdClockValue(globalClockValue);`

Reply:
my bad, sorry